plugins/devtools/src/components/Content/InfoContent/InfoContent.tsx

```diff
@@ -72,7 +73,17 @@ const copyToClipboard = ({ about }: { about: DevToolsInfo | undefined }) => {
 /** @public */
 export const InfoContent = () => {
   const classes = useStyles();
+  const [resources, setResources] = useState<string | undefined>(undefined);
```

Comment:
Might as initialize this one to `about?.resourceUtilization` right? Even though that might not currently work in practice it's kinda the correct thing to do.